src/ui/Main.java

```diff
@@ -39,6 +39,9 @@ public static void main(String[] args) {
         Tokenizer.makeTokenizer("generic_sample.txt", literals);
         Game game = new Game();
         game.parse();
-        game.evaluate();
+        System.out.println("(✿╹◡╹) ");
+        System.out.println(game.description);
+        System.out.println(game.lives);
+        System.out.println(game.title);
```

Comment:
Some quick comments to check if game was made correctly. Will eventually do this for all object creation